ddouglasz/marley-spoon-frontend
# - Setup project layout

**What does this PR do?**

- Setup project layout

 **Description of Task to be completed?**

 - The PR initializes the first step to how the project should be structured 
 - Broken into `components`, `pages` and `api`
 - As project grows, it is possible to make some changes to these structures as need arises

 **How should this be manually tested?**

 - Pulll `feat/initial-app-layout` branch, navigate through codebase to see current folder structure.
 - Fire up the app by running `yarn dev` to see the current initial page display on `localhost:3000`

 **Any background context you want to provide?**

 xxx

 **What are the relevant stories?**

 xxx

 **Screenshots (if appropriate)**

 xxx

 **Questions:**

xxx
